lib/Onyx.js

```diff
@@ -849,6 +849,7 @@ function evictStorageAndRetry(error, onyxMethod, ...args) {
  * @param {String} key
  * @param {*} value
  * @param {String} method
+ * @returns {Boolean} whether the value for the given key has changed
```

Comment:
Returning `hasChanged` from `broadcastUpdate()` changes it's responsibility a bit. 

Could we maybe pass `hasChanged` as an argument instead? That way the callers already have the value?